modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry_type_data_layout_builder.jsp

```diff
@@ -93,6 +104,19 @@ renderResponse.setTitle((fileEntryType == null) ? LanguageUtil.get(request, "new
 	</div>
 </aui:form>
 
+<liferay-frontend:component
+	componentId='<%= liferayPortletResponse.getNamespace() + "LocaleChangedHandlerComponent" %>'
+	context='<%=
```

Comment:
Can we make a method for this one too? As you did for `dlEditDDMStructureDisplayContext.getComponentContext()`?

Reply:
Hi @jeyvison 
In edit_ddm_structure_data_layout_builder.jsp, there was already the variable "DLEditDDMStructureDisplayContext" so I added the method in that class.

In this case, there is no var *Context, so I thought of adding directly. Do you think that can maintain this way? Or do you prefer to create the class and add this method?

Reply:
Hhhuimm. Got it. No, no need to create the class.